build.gradle

```diff
@@ -432,25 +432,6 @@ tasks.register("artifactDockerOnly") {
     }
 }
 
-tasks.register("generatePluginsVersion") {
```

Comment:
Docs have been changing alot, I might be out of sync.
Does this mean we no longer use reference docs workflow? - https://github.com/elastic/logstash-docs/blob/89fac69db8de43a1fca62aa1c4c983455d5986fd/.github/workflows/reference_docs-pre90.yml#L60